<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="JarvisPC" failures="1" tests="2" name="Default test(failed)" time="0.144" errors="0" timestamp="2021-11-20T06:52:32 IST">
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC4" time="0.036">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNGd_FailedTestCases.A_FailedTestCases.TC4(A_FailedTestCases.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC4 -->
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC6" time="0.006"/>
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC3" time="0.0">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC2" time="0.0">
    <ignored/>
  </testcase> <!-- TC2 -->
  <testcase classname="TestNGd_FailedTestCases.A_FailedTestCases" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Default test(failed) -->
